stable/prometheus/values.yaml

```diff
@@ -238,6 +238,13 @@ nodeExporter:
     tag: v0.14.0
     pullPolicy: IfNotPresent
 
+  ## Set update Strategy to Rolling Update instead of default OnDelete
+  ##
+  updateStrategy:
+    type: RollingUpdate
```

Comment:
Stick to the native default behavior: set `type: OnDelete` and comment the following lines.

Reply:
Fixed. I also removed the following lines and those comments will not make sense on OnDelete option.